CHANGELOG.md

```diff
@@ -54,9 +54,13 @@ For more information about this file see also [Keep a Changelog](http://keepacha
   - Modularized input record, format record, and dbfiles record into shiny modules. This allows the app to be greatly simplified to two, single-page workflows. These functions can also be used "plug-and-play" style elsewhere in PEcAn shiny apps to load in data. 
   - Replaced modularized input, format and dbfiles records with static "Ingest Workflow" page. On this page, the user can select either importing from dataONE or Uploading from local files. If creating a new format is necessary, the user can click "Create New Format" and a dropdown menu will walk them through this process. 
   - Selected files now autofill name value in input record workflow
-  - Store inputs in the global environment
+  - Store inputs and formats in the global environment
   - "Test BETY" button allows users create a record in BETY with `dbfile.input.insert`
-
+  - Removed `dbfiles.format.variables.insert` 
```

Comment:
As noted previously, you don't need to explicitly mention that you removed a function that was never part of a release. Plus, you didn't actually remove it, you renamed it.

Reply:
Not sure how this was committed as I distinctly remember deleting it. Thanks!